azure-stack/hci/manage/update-network-controller-certificates.md

```diff
@@ -372,6 +372,17 @@ To renew the Network Controller node certificate, perform the following steps on
 
 ---
 
+## Re-import certificates in Windows Admin Center
+
+If you have renewed the Network Controller REST certificate and you are using Windows Admin Center to manage SDN, you must remove the Azure Stack HCI cluster from Windows Admin Center and add it again. By doing this, you enforce Windows Admin Center to import the renewed certificate and use it for SDN management.
```

Comment:
I would use one of the two following wordings:

"By doing this, you ensure that Windows Admin Center imports the renewed certificate and uses it for SDN management."

Or

"By doing this, you force Windows Admin Center to import the renewed certificate and use it for SDN management."

Reply:
Thanks @sethmanheim. I used the first suggested blurb.